src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController_ModelComputation.cs

```diff
@@ -102,33 +102,21 @@ private void StartModelUpdateAndSelectedItemUpdateTasks(int modelUpdateDelay)
             var languageService = document.GetLanguageService<INavigationBarItemService>();
             if (languageService != null)
             {
-                // check whether we can re-use lastCompletedModel. otherwise, update lastCompletedModel here.
-                // the model should be only updated here
-                if (lastCompletedModel != null)
-                {
-                    var semanticVersion = await document.Project.GetDependentSemanticVersionAsync(CancellationToken.None).ConfigureAwait(false);
-                    if (lastCompletedModel.SemanticVersionStamp == semanticVersion && SpanStillValid(lastCompletedModel, snapshot, cancellationToken))
-                    {
-                        // it looks like we can re-use previous model
-                        return lastCompletedModel;
-                    }
-                }
```

Comment:
the semantic version is computed based on things like file versions.  but it's not really safe to use when dealing with fvrozen partial semantics as the file versions may be the same, but the overall symbol information might not be.  now that we just recompute whenever an appropriate change happens, we really don't need this.